plugins/needs_restarting.py

```diff
@@ -180,6 +188,27 @@ def run(self):
         owning_pkg_fn = functools.partial(owning_package, self.base.sack)
         owning_pkg_fn = memoize(owning_pkg_fn)
 
+        if self.opts.reboothint:
+            need_reboot = set()
+            installed = self.base.sack.query().installed()
+            for pkg in installed.filter(name=NEED_REBOOT):
+                if pkg.installtime > process_start.boot_time:
+                    need_reboot.add(pkg.name)
+            if need_reboot:
+                print('Core libraries or services have been updated '
+                      'since boot-up:')
```

Comment:
All these static strings should be marked for translation using _()

Reply:
Good point, thanks